sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/statefulOperators.scala

```diff
@@ -111,6 +111,10 @@ trait StatefulOperator extends SparkPlan {
     }
   }
 
+  lazy val stateStoreEncoding: String =
+    session.sessionState.conf.getConf(
+      SQLConf.STREAMING_STATE_STORE_ENCODING_FORMAT)
+
```

Comment:
@anishshri-db I had created a Utils object per your suggestion, but found that for some reason when we were testing, the Utils object had always just returned the fallback/default value instead of reading from the SQLConf